Jenkinsfile

```diff
@@ -46,6 +46,11 @@ git merge ${merge_opts} '${commit}' -m '${commit_msg}'
 """)
 }
 
+// https://issues.jenkins-ci.org/browse/JENKINS-37069
+def fix_workspace_label() {
+    sh "chcon -Rt svirt_sandbox_file_t '${env.WORKSPACE}'*"
```

Comment:
I should have linked the other PR where we discussed this: https://github.com/jupierce/aos-cd-jobs/pull/5

The problem is jenkins is the one adding the mount flag for the workspace, and looking through the code (linked in other PR) there's no way to change that.